syntax = "proto3";

package infgen;

message report {
  uint32 client_id = 1;
  uint32 connected = 2;
  uint32 tx_bytes = 3;
  uint32 rx_bytes = 4;
  uint32 tx_packets = 5;
  uint32 rx_packets = 6;
  message notice {
    bool online = 1;
  }
  notice note = 7;
}

message command {
  uint32 conn = 1;
  uint32 burst = 2;
  uint32 epoch = 3;
  uint64 start_ts = 4;
  uint32 setup_time = 5;
  uint32 wait_time = 6;
  uint32 duration = 7;
  uint32 length = 8;
  int32 think_time = 9;
  float ratio = 10;
  uint32 stagger_time = 11;
}

